src/Navigation/navigation.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import {Link} from 'react-router-dom';
+import './navigation.css';
+
+export default function Navigation(props){
+    return(
+        <div className='nav-panel'>
+            <Link to='/about'>
+                <NavButton 
+                    class='nav1'
+                    label='About'
+                    onClick={props.setRed}
+                />
+            </Link>
+            <Link to='/'>
+                <NavButton 
+                    class='nav2'
+                    label='Projects'
+                    onClick={props.setGreen}
+                />
+            </Link>
+            <Link to='/experience'>
+                <NavButton 
+                    class='nav3'
+                    label='Experience'
+                    onClick={props.onClick}
+                />
+            </Link>
+            <Link to='/contact'>
+                <NavButton 
+                    class='nav4'
+                    label='Contact'
+                    onClick={props.setBlue}
+                />
+            </Link>
+        </div>
+    )
+}
+
+const NavButton = (props) => {
+    return(
+        <div className={`nav-button ${props.class}`}>
+            <button type='button' className='button' onClick={props.onClick}>{props.label}</button>
```

Comment:
Another button inside an anchor - I'd recommend changing this to remove the button. You might be able to put the styles and `onClick` directly on the `Link`, but I'm not sure. `props.label` could just be a literal inside the Link - `<Link ...>Experience</Link>`